### DISCUSSION TITLE
Could not find a required file.   Name: index.js

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [mdn/mdn-community#340] [Content]
I create a website using reacjs but the file type is tsx. Whenever I try to run - yarn start, it's showing (image)-
"Could not find a required file.
  Name: index.js
  Searched in: E:\VS Code Project\portfoliov4\src
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
"  
![2023-02-27](https://user-images.githubusercontent.com/31897843/221651051-c396a838-e5ba-4426-a3ed-692e0d485a45.png)
![2023-02-27 (2)](https://user-images.githubusercontent.com/31897843/221651068-48e35877-535a-48ea-8e4e-8ada0ac28ab1.png)

Any Idea what's wrong with it?

**Comment by schalkneethling** [comment]
Hey @SANKARAMDAS, unfortunately, this is not the appropriate channel to ask for help with topics such as these. I recommend asking the question on StackOverflow or trying the Reactiflux Discord server.

https://www.reactiflux.com/
